README.md

```diff
@@ -55,6 +75,13 @@ The width (in pixels) that the image needs to be displayed in the flow.  The hei
 ## Image
 Specify how the input image will be delivered to this node.  By default the image needs to be delivered in the ```msg.payload``` of the input message.
 
+But there are a number of locations where this node can get the input image:
+1. ***msg***: image will be read from an input message.
+1. ***flow***: image will be read from in a flow variable.
```

Comment:
Err - no we don't expect nodes to get inputs from flow variables (or global) - that is "magic wiring" and is not encouraged.

Reply:
Dave,
there were already a lot of discussions on the forum, and I'm not sure anymore who likes the global/flow options and who not.  Seems you not ;-)  But I completely understand that you don't like "_magic_ wiring_", since that is the whole point of the flow ...

I had added - similar to Steve's cloned node - following options in the TypedInput:
+ 'msg'
+ 'flow'
+ 'global'
+ 'str'
+ 'bin'
Is it ok for you if I remove the flow and global, but the others remain?  I can image that 'str' (for a hardcoded base64 string image) and 'bin' (for a hardcoded buffer image) might be useful in some rare situations.  Or do you only want me to keep the 'msg' option?